simtools/model/camera.py

```diff
@@ -0,0 +1,670 @@
+
+        Returns
+        -------
+        edgePixelIndices: array_like
+            Array of edge pixel indices
+
+        '''
+
+        logger.debug('Searching for edge pixels')
+
+        edgePixelIndices = list()
+
+        for i_pix, xyPixPos in enumerate(zip(pixels['x'], pixels['y'])):
+            if pixels['funnelShape'] == 1 or pixels['funnelShape'] == 3:
+                if pixels['pixOn'][i_pix]:
+                    if len(neighbours[i_pix]) < 6:
+                        edgePixelIndices.append(i_pix)
+            elif pixels['funnelShape'] == 2:
+                if pixels['pixOn'][i_pix]:
+                    if len(neighbours[i_pix]) < 4:
+                        edgePixelIndices.append(i_pix)
+
+        return edgePixelIndices
+
+    def _plotAxesDef(self, telescopeType, plt, rotateAngle):
+        '''
+        Plot three axes definitions on the pyplot.plt instance provided.
+        The three axes are Alt/Az, the camera coordinate system and
+        the original coordinate system the pixel list was provided in.
+
+        Parameters
+        ----------
+        telescopeType: string
+            As provided by the telescope model method "TelescopeModel".
+        plt: pyplot.plt instance
+            A pyplot.plt instance where to add the axes definitions.
+        rotateAngle: float
+            The rotation angle applied
+
+        '''
+
+        invertYaxis = False
+        xLeft = 0.7  # Position of the left most axis
+        if telescopeType not in TWO_MIRROR_TELS:
+            invertYaxis = True
+            xLeft = 0.8
+
+        xTitle = r'$x_{\!pix}$'
+        yTitle = r'$y_{\!pix}$'
+        xPos, yPos = (xLeft, 0.12)
+        # The rotation of LST (above 100 degrees) raises the axes.
+        # In this case, lower the starting point.
+        if np.rad2deg(rotateAngle) > 100:
+            yPos -= 0.09
+            xPos -= 0.05
+        axesPars = {'xTitle': xTitle, 'yTitle': yTitle, 'xPos': xPos, 'yPos': yPos,
+                    'rotateAngle': rotateAngle - (1/2.)*np.pi, 'fc': 'black',
+                    'ec': 'black', 'invertYaxis': invertYaxis}
+        self._plotOneAxisDef(plt, **axesPars)
+
+        xTitle = r'$x_{\!cam}$'
+        yTitle = r'$y_{\!cam}$'
+        xPos, yPos = (xLeft + 0.15, 0.12)
+        axesPars = {'xTitle': xTitle, 'yTitle': yTitle, 'xPos': xPos, 'yPos': yPos,
+                    'rotateAngle': (3/2.)*np.pi, 'fc': 'blue',
+                    'ec': 'blue', 'invertYaxis': invertYaxis}
+        self._plotOneAxisDef(plt, **axesPars)
+
+        xTitle = 'Alt'
+        yTitle = 'Az'
+        xPos, yPos = (xLeft + 0.15, 0.25)
+        axesPars = {'xTitle': xTitle, 'yTitle': yTitle, 'xPos': xPos, 'yPos': yPos,
+                    'rotateAngle': (3/2.)*np.pi, 'fc': 'red',
+                    'ec': 'red', 'invertYaxis': invertYaxis}
+        self._plotOneAxisDef(plt, **axesPars)
+
+        return
+
+    def _plotOneAxisDef(self, plt, **axesPars):
```

Comment:
I think it's a good practice in python to always use the name kwargs for **kwargs. But that is only an impression, I'm not sure about it.

Reply:
Shame, I really like the fact the dictionary provides information on what's inside. But OK, rules are rules, I changed it to kwargs.